### ISSUE TITLE
Clean up code to remove old calendar logic and references

### ISSUE THREAD

**Issue by JonathanCrd** [Azure/azure-sdk-tools#7142]
We have old code from the first ever calendar widget in the Scheduling Tool (Unused variables, commented code, hidden controls, etc.)

**Comment by maririos** [comment]
I removed the previous views and updated the tests

**Comment by maririos** [comment]
@JonathanCrd do you think there are more things we need to delete?

**Comment by JonathanCrd** [comment]
Yes, I haven't take a good amount of time to inspect the whole code but I know there are collections and variables we don't need anymore. The goal with this issue would be to delete all of that, not only the direct references to the old calendar.
